hebi_a-2085-06_moveit_config/config/A-2085-06.srdf

```diff
@@ -10,59 +10,66 @@
+    <disable_collisions link1="Arm/J4_wrist1/INPUT_INTERFACE" link2="Arm/J5_wrist2/INPUT_INTERFACE" reason="Never" />
+    <disable_collisions link1="Arm/J4_wrist1/INPUT_INTERFACE" link2="Arm/J6_wrist3/INPUT_INTERFACE" reason="Never" />
+    <disable_collisions link1="Arm/J4_wrist1/INPUT_INTERFACE" link2="elbow_wrist1/INPUT_INTERFACE" reason="Adjacent" />
+    <disable_collisions link1="Arm/J4_wrist1/INPUT_INTERFACE" link2="end_effector/INPUT_INTERFACE" reason="Never" />
+    <disable_collisions link1="Arm/J4_wrist1/INPUT_INTERFACE" link2="shoulder_bracket/INPUT_INTERFACE" reason="Never" />
+    <disable_collisions link1="Arm/J4_wrist1/INPUT_INTERFACE" link2="shoulder_elbow/INPUT_INTERFACE" reason="Never" />
+    <disable_collisions link1="Arm/J4_wrist1/INPUT_INTERFACE" link2="wrist2_bracket/INPUT_INTERFACE" reason="Adjacent" />
+    <disable_collisions link1="Arm/J4_wrist1/INPUT_INTERFACE" link2="wrist3_bracket/INPUT_INTERFACE" reason="Never" />
+    <disable_collisions link1="Arm/J5_wrist2/INPUT_INTERFACE" link2="Arm/J6_wrist3/INPUT_INTERFACE" reason="Never" />
+    <disable_collisions link1="Arm/J5_wrist2/INPUT_INTERFACE" link2="elbow_wrist1/INPUT_INTERFACE" reason="Never" />
+    <disable_collisions link1="Arm/J5_wrist2/INPUT_INTERFACE" link2="end_effector/INPUT_INTERFACE" reason="Never" />
+    <disable_collisions link1="Arm/J5_wrist2/INPUT_INTERFACE" link2="shoulder_bracket/INPUT_INTERFACE" reason="Never" />
+    <disable_collisions link1="Arm/J5_wrist2/INPUT_INTERFACE" link2="wrist2_bracket/INPUT_INTERFACE" reason="Adjacent" />
+    <disable_collisions link1="Arm/J5_wrist2/INPUT_INTERFACE" link2="wrist3_bracket/INPUT_INTERFACE" reason="Adjacent" />
+    <disable_collisions link1="Arm/J6_wrist3/INPUT_INTERFACE" link2="end_effector/INPUT_INTERFACE" reason="Adjacent" />
+    <disable_collisions link1="Arm/J6_wrist3/INPUT_INTERFACE" link2="wrist2_bracket/INPUT_INTERFACE" reason="Never" />
+    <disable_collisions link1="Arm/J6_wrist3/INPUT_INTERFACE" link2="wrist3_bracket/INPUT_INTERFACE" reason="Adjacent" />
+    <disable_collisions link1="elbow_wrist1/INPUT_INTERFACE" link2="end_effector/INPUT_INTERFACE" reason="Never" />
     <disable_collisions link1="elbow_wrist1/INPUT_INTERFACE" link2="shoulder_elbow/INPUT_INTERFACE" reason="Never" />
-    <disable_collisions link1="elbow_wrist1/INPUT_INTERFACE" link2="wrist1_wrist2/INPUT_INTERFACE" reason="Never" />
-    <disable_collisions link1="elbow_wrist1/INPUT_INTERFACE" link2="wrist2_wrist3/INPUT_INTERFACE" reason="Never" />
+    <disable_collisions link1="elbow_wrist1/INPUT_INTERFACE" link2="wrist2_bracket/INPUT_INTERFACE" reason="Never" />
+    <disable_collisions link1="elbow_wrist1/INPUT_INTERFACE" link2="wrist3_bracket/INPUT_INTERFACE" reason="Never" />
+    <disable_collisions link1="end_effector/INPUT_INTERFACE" link2="wrist2_bracket/INPUT_INTERFACE" reason="Never" />
+    <disable_collisions link1="end_effector/INPUT_INTERFACE" link2="wrist3_bracket/INPUT_INTERFACE" reason="Never" />
```

Comment:
EE again.  I'm assuming this must be on purpose, b/c it is consistent and the setup assistant wouldn't generate something invalid here.